glib/src/gobject/binding_group.rs

```diff
@@ -140,21 +140,23 @@ impl<'a> BindingGroupBuilder<'a> {
             user_data: ffi::gpointer,
         ) -> ffi::gboolean {
             let transform_data =
-                &*(user_data as *const (TransformFn, TransformFn, ParamSpec, ParamSpec));
+                &*(user_data as *const (TransformFn, TransformFn, Option<ParamSpec>, ParamSpec));
```

Comment:
I don't think this will work like this. You need the source pspec here for checking that the value types are matching, but if there is none duration construction then you can't really get it in here easily.

Reply:
This should probably just get rid of the `ParamSpec`s and use strings and `find_property`, that is what the `BindingGroup` does internally

Reply:
Sounds like a plan